app/templates/events/import.hbs

```diff
@@ -1 +1,2 @@
-import data
+{{events/event-import-section}}
```

Comment:
And lets push this content one level down by pushing the `{{outlet}}` at `templates/events.hbs` down one more level.